src/components/card/CommentItem.vue

```diff
@@ -48,7 +48,9 @@
 				<NcActionButton icon="icon-close" @click="hideUpdateForm" />
 			</NcActions>
 			<div class="spacer" />
-			<div class="timestamp">
+			<div class="timestamp"
+				:aria-label="formattedTimestamp"
+				:title="'Created: ' + formattedTimestamp">
```

Comment:
```suggestion
				:title="formattedTimestamp">
```

Combined with the below suggestion to have a translated Created string